zio-cache/shared/src/main/scala/zio/cache/Cache.scala

```diff
@@ -1,29 +1,24 @@
 package zio.cache
 
 import zio.internal.MutableConcurrentQueue
-import zio.{Exit, IO, Promise, UIO, URIO, ZIO}
+import zio.{Exit, IO, Promise, Ref, UIO, URIO, ZIO}
 
 import java.time.{Duration, Instant}
 import java.util.Map
 import java.util.concurrent.atomic.{AtomicBoolean, LongAdder}
 
 /**
- * A `Cache` is defined in terms of a lookup function that, given a key of
- * type `Key`, can either fail with an error of type `Error` or succeed with a
- * value of type `Value`. Getting a value from the cache will either return
- * the previous result of the lookup function if it is available or else
- * compute a new result with the lookup function, put it in the cache, and
- * return it.
+ * A `Cache` is defined in terms of a lookup function that, given a key of type `Key`, can either fail with an error of
```

Comment:
Sure thing. I upgraded scalafmt but the latest version does some funky stuff. I actually had to downgrade it back to 3.0.3 (instead of the latest 3.0.5).... Anyhow, I will figure out the format config for comments and bring the old style back :)